cmd/juju/application/deployer/bundlehandler.go

```diff
@@ -1439,8 +1437,196 @@ func resolve(placeholder string, results map[string]string) string {
 	return results[id]
 }
 
+// ModelExtractor provides everything we need to build a
+// bundlechanges.Model from a model API connection.
+type ModelExtractor interface {
+	GetAnnotations(tags []string) ([]params.AnnotationsGetResult, error)
+	GetConstraints(applications ...string) ([]constraints.Value, error)
+	GetConfig(branchName string, applications ...string) ([]map[string]interface{}, error)
+	Sequences() (map[string]int, error)
+}
+
+func buildModelRepresentation(
```

Comment:
BuildModelRepresentation and the ModelExtactor interface were moved to https://github.com/juju/juju/blob/f76391aee8f3ce5eb07eaddd8397464f5d48f3e8/cmd/juju/application/bundle/bundle.go#L28 in 2.9. To be used for deploy and bundlediff.  The small change from the PR can be made there.

I do not believe that the content was changed, just the location.